libvdrtools/indy-api-types/Cargo.toml

```diff
@@ -13,15 +13,15 @@ rust-base58 = ["bs58"]
 
 [dependencies]
 failure = "0.1.8"
-futures = { version = "0.3", default-features = false }
+futures = { version = "0.3", default-features = false, features = ["std"] }
```

Comment:
why needed?

Reply:
Same reason as above: `indy-api-types` fails to build without it. Actually, thanks to your comment, I found out the piece of code using the feature seems to have been dead, so I removed it along with the feature.